docs/api/components/schemas.yaml

```diff
@@ -2162,6 +2165,46 @@ Startup-config:
       type: integer
       readOnly: true
   title: Startup-config
+Runtime-config:
+  type: object
+  properties:
+    logging:
```

Comment:
Is it correct to add all these log settings here, or as part of this review?

Reply:
They are part of the runtime config endpoint request schema. In order to add the fields I needed I had to create docs from the runtime endpoint insteead of sharing the docs from the boostrap config.

Reply:
My fault, I was thinking this was db/_config.